src/main/flight/imu.c

```diff
@@ -491,6 +495,19 @@ float getCosTiltAngle(void)
     return rMat[2][2];
 }
 
+int getHeadingDirection(){
```

Comment:
Betaflight uses K&R style: Function definitions have the opening bracket on a new line.